### ISSUE TITLE
wiztree@4.00: download via aria2 failed

### ISSUE THREAD

**Issue by redactedscribe** [lukesampson/scoop-extras#6326]
```
ERROR Download failed! (Error 22) HTTP response header was bad or unexpected
ERROR https://wiztreefree.com/files/wiztree_4_00_portable.zip
    referer=https://wiztreefree.com/files/
    dir=D:\Scoop\cache
    out=wiztree#4.00#https_wiztreefree.com_files_wiztree_4_00_portable.zip
ERROR & 'D:\Scoop\apps\aria2\current\aria2c.exe' --input-file='D:\Scoop\cache\wiztree.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.1 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=12900 --continue --summary-interval 0
```

From https://www.diskanalyzer.com/whats-new:
```
- The wiztree web site has moved to from "wiztreefree.com" to "diskanalyzer.com"
```

After setting `scoop config aria2-enabled false`, the download works.

**Comment by allenatdecisiv** [comment]
I can confirm turning off aria2 worked for me.

**Comment by psydvl** [comment]
Same to me
<details>

```
Updating 'wiztree' (3.41 -> 4.00)
Downloading new version
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: b870a4|ERR |       0B/s|C:/Users/psydv/scoop/cache/wiztree#4.00#https_wiztreefree.com_files_wiztree_4_00_portable.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 22) HTTP response header was bad or unexpected
ERROR https://wiztreefree.com/files/wiztree_4_00_portable.zip
    referer=https://wiztreefree.com/files/
    dir=C:\Users\psydv\scoop\cache
    out=wiztree#4.00#https_wiztreefree.com_files_wiztree_4_00_portable.zip

ERROR & 'C:\Users\psydv\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\psydv\scoop\cache\wiztree.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.1 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=3944 --continue --summary-interval 0
```

</details>

Error with `aria2` only

**Comment by AntonOks** [comment]
fixed with #6328 ?!!
